Multisig/Logic/AuthenticationController.swift

```diff
@@ -102,6 +109,29 @@ class AuthenticationController {
         }
         return Data(derivedKey).toHexString()
     }
+
+
+    // MARK: - Biometry
+
+    /// Is device hardware supports the biometry
+    var isBiometricsSupported: Bool {
```

Comment:
I would be for adding it to the lib, but okay :)

Reply:
will move it after this is merged